util/linux_install.sh

```diff
@@ -75,13 +75,10 @@ elif grep ID /etc/os-release | grep -q 'arch\|manjaro'; then
 elif grep ID /etc/os-release | grep -q gentoo; then
 	echo GENTOO_WARNING | fmt
 	echo -n "Proceed (y/N)? "
```

Comment:
```suggestion
	printf "Proceed (y/N)? "
```
Replace non-POSIX `echo -n` with `printf`